### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by tinkeringtuck** [RobertWojtowicz/export2garmin#47] [Q&A]
Apologies, if I was more knowledgeable on Git, I think this answer would be intuitive but what is the recommended way to receive updates as the project is updated? 

Currently, I have cloned the repo and have just made changes locally to the necessary files for it to work in my application. I have not forked the repository or bothered making any local git commits. Would you recommend a different approach all together or is this as you intend, and I just `git pull` changes as they are made, merging as necessary? 

Thanks.

**Comment by tinkeringtuck** [comment]
@RobertWojtowicz Thanks for yet another release today. Appreciate your commitment to this project. Any thoughts here on how we can best stay up to date with your continued efforts? Thanks.

**Reply by RobertWojtowicz** [reply]
Hi

I don't have an answer to this, because as I make thick changes, I clean the repository from time to time I like its transparency and don't want to put away old versions.

Now will be preparing such a version, because I am making quite a big change, adding support for the BLE adapter not only by HCI number, but by the correct MAC, because as there are many adapters, these HCI numbers like to float each other.

After that, the repository is more likely to have small changes and will be cleaned less often.

**Comment by RobertWojtowicz** [comment]
@tinkeringtuck 
I have released a new version: https://github.com/RobertWojtowicz/export2garmin/releases

BR,
Robert
